packages/esm-appointments-app/src/hooks/useAppointmentsCalendar.ts

```diff
@@ -1,36 +1,44 @@
-import { openmrsFetch, type OpenmrsResource } from '@openmrs/esm-framework';
+import { openmrsFetch } from '@openmrs/esm-framework';
 import dayjs from 'dayjs';
 import useSWR from 'swr';
 import { omrsDateFormat } from '../constants';
+import { type DailyAppointmentsCountByService } from '../types';
 
-interface CalendarEvent {
-  services: Array<{
-    appointmentServiceName: string;
-    allAppointmentsCount: number;
-    missedAppointmentsCount: number | null;
-    appointmentDate: string | null;
-  }>;
-  appointmentDate: string;
-  appointmentService: string;
-  appointmentCountMap: OpenmrsResource | null;
-  totalServiceCount: number;
+interface AppointmentCountMapEntry {
+  allAppointmentsCount: number;
+}
+
+interface AppointmentSummaryResponse {
+  appointmentService: {
+    name: string;
+  };
+  appointmentCountMap: Map<string, AppointmentCountMapEntry>;
 }
 
 export const useAppointmentsCalendar = (forDate: string, period: string) => {
   const { startDate, endDate } = evaluateAppointmentCalendarDates(forDate, period);
-  const url = `/ws/rest/v1/appointment/appointmentCalendar?startDate=${startDate}&endDate=${endDate}`;
+  const url = `/ws/rest/v1/appointment/appointmentSummary?startDate=${startDate}&endDate=${endDate}`;
```

Comment:
This and the "results" code block below is the meat of this code, basically we are switching from using the "appointmentCalender" endpoint, which is only a custom fork of the Bahmni Appointment module, to use the appointmentSummary endpoint that is in the released module.